source/_components/switch.rainmachine.markdown

```diff
@@ -13,55 +13,31 @@ ha_iot_class: "Cloud Polling"
 ha_release: 0.51
 ---
 
-The `rainmachine` switch platform allows you to control programs and zones within a [RainMachine smart Wi-Fi sprinkler controller](http://www.rainmachine.com/).
+The `rainmachine` switch platform allows you to control programs and zones
+within a [RainMachine smart Wi-Fi sprinkler controller](http://www.rainmachine.com/). 
 
-## {% linkable_title Configuring the Platform %}
+<p class='note'>
+You must have the [RainMachine component](https://www.home-assistant.io/components/rainmachine/)
+configured to use this switch. After configuring that component, switches will
+automatically appear.
+</p>
 
-The platform allows for either local (i.e., directly across the LAN) or remote (i.e., through RainMachine's cloud API) access; the route you choose will dictate what your configuration should look like.
+## {% linkable_title Configuring the Platform %}
 
-For local access, specify the IP address/hostname of your RainMachine unit, your RainMachine password, and optionally, the device's HTTP port:
+To optionally extend the functionality of these switches, augment your 
+`configuration.yaml` as below:
 
 ```yaml
-switch:
-  platform: rainmachine
+rainmachine:
   ip_address: 192.168.1.100
   password: YOUR_PASSWORD
-```
-
-For remote access, specify your RainMachine username/email and password:
-
-```yaml
-switch:
-  platform: rainmachine
-  email: user@host.com
-  password: YOUR_PASSWORD
+  switches:
+    zone_run_time: 240
 ```
 
 {% configuration %}
```

Comment:
Remove this.

Reply:
Why? This follows a model similar to the Nest docs: https://www.home-assistant.io/components/sensor.nest/